src/lint_errors.cc

```diff
@@ -32,6 +32,18 @@
 
 namespace zetasql::linter {
 
+std::map<std::string, ErrorCode> GetErrorMap() {
```

Comment:
Good idea to make a unit test that verifies this map contains string names for all checks. This way it will be hard to forget adding a new one :)

Reply:
This was the one of the reasons I made CheckList :) I will definitely add it to unit test.